clients/naming_client/naming_cache/service_info_holder.go

```diff
@@ -44,9 +44,8 @@ func NewServiceInfoHolder(namespace, cacheDir string, updateCacheWhenEmpty, notL
 		subCallback:          NewSubscribeCallback(),
 		UpdateTimeMap:        cache.NewConcurrentMap(),
 	}
+	serviceInfoHolder.ServiceInfoMap = cache.NewConcurrentMap()
```

Comment:
Initialize the ServiceInfoMap when you build ServiceInfoHolder，maybe it's better